Content.Server/Explosion/EntitySystems/TriggerSystem.cs

```diff
@@ -76,6 +81,53 @@ public override void Initialize()
             SubscribeLocalEvent<ExplodeOnTriggerComponent, TriggerEvent>(HandleExplodeTrigger);
             SubscribeLocalEvent<FlashOnTriggerComponent, TriggerEvent>(HandleFlashTrigger);
             SubscribeLocalEvent<GibOnTriggerComponent, TriggerEvent>(HandleGibTrigger);
+
+            SubscribeLocalEvent<AnchorOnTriggerComponent,           TriggerEvent>(HandleAnchorTrigger);
+            SubscribeLocalEvent<SoundOnTriggerComponent,            TriggerEvent>(HandleSoundTrigger);
+            SubscribeLocalEvent<PointLightEnableOnTriggerComponent, TriggerEvent>(HandlePointLightTrigger);
+            SubscribeLocalEvent<GravityWellOnTriggerComponent,      TriggerEvent>(HandleGravityWellTrigger);
+            SubscribeLocalEvent<SingularityDistortionOnTriggerComponent, TriggerEvent>(HandleDistortionTrigger);
```

Comment:
Don't worry about indenting it as it won't be consistent anyway.

Method names also shouldn't be prefixed by "Handle" anymore, it should be "On".